src/ripple/rpc/tests/JSONRPC.test.cpp

```diff
@@ -1014,21 +1014,21 @@ R"({
 { "Malformed Account in submit_multisigned.",
 R"({
     "command": "submit_multisigned",
-    "Signers": [
-        {
-            "Signer": {
-                "Account": "rPcNzota6B8YBokhYtcTNqQVCngtbnWfux",
-                "TxnSignature": "3045022100F9ED357606932697A4FAB2BE7F222C21DD93CA4CFDD90357AADD07465E8457D6022038173193E3DFFFB5D78DD738CC0905395F885DA65B98FDB9793901FE3FD26ECE",
-                "SigningPubKey": "02FE36A690D6973D55F88553F5D2C4202DE75F2CF8A6D0E17C70AC223F044501F8"
-            }
-        }
-    ],
     "tx_json": {
         "Account": "NotAnAccount",
         "Amount": "1000000000",
         "Destination": "rnUy2SHTrB9DubsPmkJZUXTf5FcNDGrYEA",
         "Fee": 50,
         "Sequence": 0,
+        "Signers": [
+            {
+                "Signer": {
+                    "Account": "rPcNzota6B8YBokhYtcTNqQVCngtbnWfux",
+                    "TxnSignature": "3045022100F9ED357606932697A4FAB2BE7F222C21DD93CA4CFDD90357AADD07465E8457D6022038173193E3DFFFB5D78DD738CC0905395F885DA65B98FDB9793901FE3FD26ECE",
+                    "SigningPubKey": "02FE36A690D6973D55F88553F5D2C4202DE75F2CF8A6D0E17C70AC223F044501F8"
+                }
+            }
+        ],
         "SigningPubKey": "",
```

Comment:
I'll be improving the situation by inserting it automatically in `sign_for` (code review to come).  But, yeah, if someone doesn't use `sign_for` they must remember to include the field.